nimrod/tests/test_nimrod.py

```diff
@@ -1,4 +1,4 @@
-import os
```

Comment:
Why did you comment this code? I though the tag @unittest.skip would skip the execution of the tagged test cases.

Reply:
My bad. I had commented this file when I was adding CI in our repository (we doesn't had the tag at that time), and then I forgot to remove when did the merge.